### ISSUE TITLE
[Starfield] Missing Properties button for Papyrus fragments in Quests and TerminalMenu forms

### ISSUE THREAD

**Issue by DownfallNemesis** [Perchik71/Creation-Kit-Platform-Extended#80]
Tested in 0.4-822 and 0.4-829
The following buttons are missing:
- Compile
- Properties
- Edit
Screenshots of the missing buttons attached below.
![Screenshot 2024-11-24 173423](https://github.com/user-attachments/assets/b6110f0b-ce40-4fa5-94fa-354f8d6032c9)
![Screenshot 2024-11-24 173448](https://github.com/user-attachments/assets/9cd2d7e3-cd04-428c-b38c-8ddfd9bb7226)

## How it looks in vanilla CK
![Screenshot 2024-11-24 181828](https://github.com/user-attachments/assets/76216838-2e19-4ae6-8ac7-80646d0c0bf0)

**Comment by DJLegends1011** [comment]
same as [#68 ](https://github.com/Perchik71/Creation-Kit-Platform-Extended/issues/68)

**Comment by kristoferblack** [comment]
+1 on this. Pretty major issue for anyone touching anything with Papyrus fragments.

**Comment by yak3d** [comment]
Looks like its there, but the tab is too small to show it.

![image](https://github.com/user-attachments/assets/74e5c45f-19b0-4634-93f4-45ef35a950bd)

You can tab to them, although you're doing it blindly but that can serve as a workaround for now. Going to look at this tonight over the Properties window issue.

**Comment by yak3d** [comment]
#83 can fix this by shrinking the text box a little and shifting the buttons up. But it still needs more time, #83 is just to unblock folks.

![image](https://github.com/user-attachments/assets/ce4f1282-ab54-4d9c-a587-f22441e64e39)

**Comment by Perchik71** [comment]
@yak3d only thing is that the style of the Properties button has disappeared, it has become a regular button

**Comment by yak3d** [comment]
> @yak3d only thing is that the style of the Properties button has disappeared, it has become a regular button

hmm will take a look

**Comment by yak3d** [comment]
pretty sure i know the cause. will push a fix asap.

**Comment by ZeeOgre** [comment]
It appears that at least for my 5K wide, it's still a problem:

![image](https://github.com/user-attachments/assets/49aeb5e4-2c15-4698-98d9-99c47542a524)


![image](https://github.com/user-attachments/assets/28f45e33-5f72-4d41-89d0-2d9273bd1a3f)

**Comment by yak3d** [comment]
Yeah not crazy surprised. This still needs a proper fix as resizing stuff was just a stopgap :/ I've been looking into it more still.

**Comment by Perchik71** [comment]
This will be a problem, since the size of the target window is always constant, so scaling by this does not work.
